### ISSUE TITLE
Activating Gitpod?

### ISSUE THREAD

**Issue by naoe-tatara** [uio-carpentry/uio-carpentry.github.io#43]
Gitpod is quite handy, which enable editing on a browser as well as previewing.
With a fork on my account, I tested Gitpod with the following files in [a branch](https://github.com/naoe-tatara/uio-carpentry.github.io/tree/naoe-tatara/gitpod-setup):

```code 
# file: .gitpod.Dockerfile

FROM gitpod/workspace-full

USER root

RUN apt-get update && apt-get install -yq hugo
```

```code 
# file: .gitpod.yml

image:
  file: .gitpod.Dockerfile

tasks:
  - command: 'hugo server --baseUrl $(gp url 1313) --appendPort=false'

ports:
  - port: 1313
    onOpen: open-preview
```
(ref: [Using Hugo on Gitpod](https://napjose.ph/posts/using-hugo-on-gitpod/) )

I wonder if it helps us editing without needing installing Go and Hugo locally. If this sounds reasonable, I would send a PR from the fork/branch above to this upstream/main.

**Comment by Teebusch** [comment]
@naoe-tatara  Interesting! Definitely a cool tool! I womder i f this might be overkill? [VSCode has a decent Github integration](https://code.visualstudio.com/docs/editor/github). Once you clone the repo and install Go and Hugo you are all set.  But if  you would find it useful and can get it to work, please go ahead. It seems like it requires minimal changes to the repo. A dockerfile and a yaml.

 Are you thinking of using this for content editing mainly? In case you just want a nicer mardown preview / editor,  a CMS like [forestry.io](https://forestry.io/) could be a better choice.

**Comment by naoe-tatara** [comment]
Thanks @Teebusch for the suggestions! I did not know about the Forestry. I am using VSCode for editing but have not used the extension. Thank you for sharing!
 
My first intention was to make threshold for editing lower for later generations but still enabling preview before committing. However now I feel like it is probably best to stick to the basics with making it possible to work on local, push to one's fork and send a PR to this, because I got a bit unsure if the Gitpod goes fine in our case where the website reads in contents from other repositories as well... Forestry says that one can have up to 3 sites for personal account with free plan, so maybe we can keep this backhead and introduce it if it seems necessary.
